src/singletons/Theme.cpp

```diff
@@ -422,13 +463,29 @@ std::optional<ThemeDescriptor> Theme::findThemeByKey(const QString &key)
     return std::nullopt;
 }
 
-void Theme::parseFrom(const QJsonObject &root)
+void Theme::parseFrom(const QJsonObject &root, bool isCustomTheme)
 {
-    parseColors(root, *this);
-
     this->isLight_ =
         root["metadata"_L1]["iconTheme"_L1].toString() == u"dark"_s;
 
+    std::optional<QJsonObject> fallbackTheme;
+    if (isCustomTheme)
+    {
+        // Only attempt to load a fallback theme if the theme we're loading is a custom theme
+        auto fallbackThemeName =
+            root["metadata"_L1]["fallbackTheme"_L1].toString("Dark");
```

Comment:
This should probably fall back to "Light" if `this->isLight_` and "Dark" otherwise.

Reply:
Should be resolved by [`e7d98e6` (#5198)](https://github.com/Chatterino/chatterino2/pull/5198/commits/e7d98e6397400d48b1c949d9dc88115b1c627bcc)

The reason I did not implement this in the first place is that themes don't have a concept of "dark" or "light" themes really, just which color icons they use, which is the opposite of dark/light theme which felt spaghetti. But I'm fine with this solution